.github/workflows/check-devbox.yaml

```diff
@@ -0,0 +1,34 @@
+name: Check Devbox
+run-name: Check Devbox - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  pull_request:
+    paths:
+      - 'devbox.json'
+      - 'devbox.lock'
+      - 'build.assets/flake/**'
+  merge_group:
+    paths:
+      - 'devbox.json'
+      - 'devbox.lock'
+      - 'build.assets/flake/**'
+
+jobs:
+  check-devbox:
+    if: ${{ !startsWith(github.head_ref, 'dependabot/') }}
+    name: Check Devbox
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install devbox
+        run: curl -fsSL https://get.jetpack.io/devbox | FORCE=1 bash
```

Comment:
Could we do the `curl | bash` before we checkout (to descope source code)?

I'm thinking about attacks like https://about.codecov.io/security-update/ here.

I'm also somewhat amused that we're using a _floating_ version of devbox, when the entire point of the tool is to pin the toolchain to specific known versions.  We should probably pin the devbox version too. 😄